inlang/source-code/sdk/src/defaultProjectSettings.ts

```diff
@@ -0,0 +1,27 @@
+import type { ProjectSettings } from "@inlang/project-settings"
+
+/**
+ * Default project settings for createNewProject
+ * from paraglide-js/src/cli/commands/init/defaults.ts
+ */
+export const defaultProjectSettings = {
+	$schema: "https://inlang.com/schema/project-settings",
+	sourceLanguageTag: "en",
+	languageTags: ["en"],
+	modules: [
+		// for instant gratification, we're adding common rules
+		"https://cdn.jsdelivr.net/npm/@inlang/message-lint-rule-empty-pattern@latest/dist/index.js",
+		"https://cdn.jsdelivr.net/npm/@inlang/message-lint-rule-missing-translation@latest/dist/index.js",
+		"https://cdn.jsdelivr.net/npm/@inlang/message-lint-rule-without-source@latest/dist/index.js",
+		"https://cdn.jsdelivr.net/npm/@inlang/message-lint-rule-valid-js-identifier@latest/dist/index.js",
+
+		// default to the message format plugin because it supports all features
+		"https://cdn.jsdelivr.net/npm/@inlang/plugin-message-format@latest/dist/index.js",
+
+		// the m function matcher should be installed by default in case Sherlock (VS Code extension) is adopted
+		"https://cdn.jsdelivr.net/npm/@inlang/plugin-m-function-matcher@latest/dist/index.js",
+	],
+	"plugin.inlang.messageFormat": {
+		pathPattern: "./messages/{languageTag}.json",
+	},
```

Comment:
I recommend to use `./plugin.inlang.messageFormat/messages/{languageTag}.json` instead. 

- generally: namespace plugin "storage" to avoid clashes with features we will introduce
- in this case: we will very likely want to use `/messages` for persistency

Reply:
sounds good to me.

Reply:
We should also consider adding a version identifier to the path - we discussed this earlier - we might need to migrate.

Reply:
@martin-lysk do you have link to the earlier discussion?

Versioning directory paths is a bit too in-your-face for my taste (e.g. when paths show up in git logs, diffs, etc)
Why can't we rely on the schema refs inside the files?

Reply:
> @jldec Are you saying we should default to ./plugin.inlang.messageFormat/{languageTag}.json or are you saying that the current default ./messages/{languageTag}.json is fine?

Change the path pattern to `./plugin.inlang.messageFormat/{languageTag}.json` to namespace plugin data and avoid conflicts. 

> Again, I'm a bit wary of adding long folder paths into the main tree of users' repos without a really good reason.

Dont forget that users won't see the inside of a `.inlang` file in the long run. I see a good reason here to establish a good practice of namespacing plugins to avoid conflicts.

Reply:
@samuelstroschein I talked with @martin-lysk and I think I understand your comment:
- for this PR the messages should live at `./project.inlang/plugin.inlang.messageFormat/{languageTag}.json`
- for non-plugin based persistence, we should reserve paths starting with `./project.inlang/messages/...`

Reply:
BTW - at the time of this writing the [default settings](https://github.com/opral/monorepo/blob/d573ab8c43ee27c4be14e6e2561a9eb6364f18af/inlang/source-code/sdk/src/defaultProjectSettings.ts#L25) had `pathPattern: "./messages/{languageTag}.json",` which results in a tree like this:

 ```
.
├── messages
│   ├── de.json
│   └── en.json
├── project.inlang
│   └── settings.json
```

Reply:
options to resolve the issue with relative pathpattern pointing ☝️ 

1. leave /messages outside the project for this PR
2. use a magic prefix in the pathattern like `$/...`
3. use a different key like `projectpath`: instead of `pathpattern`: and use that to populate pathpattern at runtime
4. change the meaning of pathpattern and break/migrate all existing projects 😬 
5. write the project `{name}.inlang` into the pathpattern 😬 

@martin-lysk @samuelstroschein wdyt?

Reply:
prob 1 then.

Reply:
I thought about this Option 1 makes sense anyway since this pr creates a default config with a plugin **not** with default persistence. Message plugins currently write to and read from a folder outside of the .inlang folder. When we settled on a format for the message during persistence and provide a persistence mechanism we don't migrate to the persistence we will load the message using the plugin (and we rename the method loadMessages - to import)

Reply:
Thanks for the feedback - will go with 1 then for this PR.

Reply:
https://github.com/opral/inlang-message-sdk/issues/34 (MESDK-70)